lgsm/functions/check_deps.sh

```diff
@@ -185,7 +185,7 @@ if [ "${javacheck}" == "1" ]; then
 		# Define required dependencies for SteamCMD.
 		if [ "${appid}" ]; then
 			# lib32gcc1 is now called lib32gcc-s1 in debian 11
-			if [ "${distroid}" == "debian" ]&&[ "${distroversion}" == "11" ]; then
+			if { [ "${distroid}" == "debian" ]&&[ "${distroversion}" == "11" ]; }||{ [ "${distroid}" == "ubuntu" ]&&[ "${distroversion}" == "20.10" ]; }; then
```

Comment:
not sure why this `{}` are here

Reply:
onyl because the shellcheck complained and it mentioned it is faster

Reply:
never seen that before

Reply:
Are you able to point me to the SC number so I can take a look as this diverges from the current standard? Thanks

Reply:
SC2235

Reply:
the reason I haven't come across this befoe is because I haven't of an if statement with both `&&` and `||` in it. 😄